cohttp-eio/src/body.ml

```diff
@@ -250,3 +250,19 @@ let write_body writer body =
   | Chunked chunk_writer -> write_chunked writer chunk_writer
   | Custom f -> f writer
   | Empty -> ()
+
+let add_content_length (requires_content_length : unit -> bool) headers body :
```

Comment:
Maybe I should ask it here, why do you need `requires_content_length : unit -> bool` instead of simply `bool`? You are going to call the function in any case

Reply:
Indeed. This is just a `bool` now.